src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/InPrisonInterestsEntity.kt

```diff
@@ -42,12 +46,10 @@ class InPrisonInterestsEntity(
   @field:NotNull
   var reference: UUID? = null,
 
-  @OneToMany(cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
-  @JoinColumn(name = "interests_id", nullable = false)
+  @OneToMany(mappedBy = "parent", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.LAZY)
```

Comment:
The same changes have been applied to the other entities, so these can be skipped...